### ISSUE TITLE
[SUGGESTION] Request to add the ability to uninstall certain projects in the solution

### ISSUE THREAD

**Issue by silmoonsone** [microsoft/vscode-dotnettools#1156]
### Describe the feature you'd like

Request to add the ability to uninstall certain projects in the solution:
We need the ability to uninstall projects similar to that in Visual Studio.
Usually a solution contains many projects, and there is a requirement not to build or compile some projects.

**Comment by AbhitejJohn** [comment]
@silmoonsone : Thanks for letting us know. Did you mean exclude/unload projects from the solution?

**Comment by silmoonsone** [comment]
Yes, my solution contains some .Net frameworks projects that cannot be compiled in VSCode.
These uncompiled projects have no reference relationship with my main cross-platform development work, they are just independent existences, I just need to unload them.
I am unlikely to delete them from my solution, the reason for keeping them is that this project is developed in both Visual Studio and VSCode.

**Comment by silmoonsone** [comment]
> @silmoonsone : Thanks for letting us know. Did you mean exclude/unload projects from the solution?

Yes, my solution contains some .Net frameworks projects that cannot be compiled in VSCode.
These uncompiled projects have no reference relationship with my main cross-platform development work, they are just independent existences, I just need to unload them.
I am unlikely to delete them from my solution, the reason for keeping them is that this project is developed in both Visual Studio and VSCode.

**Comment by silmoonsone** [comment]
> @silmoonsone : Thanks for letting us know. Did you mean exclude/unload projects from the solution?

@AbhitejJohn 

Yes, my solution contains some .Net frameworks projects that cannot be compiled in VSCode.
These uncompiled projects have no reference relationship with my main cross-platform development work, they are just independent existences, I just need to unload them.
I am unlikely to delete them from my solution, the reason for keeping them is that this project is developed in both Visual Studio and VSCode.
